src/ucp/rndv/rndv_rkey_ptr.c

```diff
@@ -94,6 +94,12 @@ ucp_proto_rndv_rkey_ptr_init(const ucp_proto_init_params_t *init_params)
     return status;
 }
 
+static ucp_lane_map_t
+ucp_rndv_rkey_ptr_query_lane_map(const ucp_proto_rndv_rkey_ptr_priv_t *rpriv)
```

Comment:
maybe pass ucp_proto_query_attr_t*attr for this function to fill?
also could rename it to ucp_rndv_rkey_ptr_query() and let it call ucp_proto_default_query as "base implementation"

Reply:
Addressed by https://github.com/openucx/ucx/pull/8986/commits/d3fce5173f95272b332ec2688ec80b787a947e4b